src/emulator.cpp

```diff
@@ -58,10 +58,11 @@ Frontend::WindowSDL* g_window = nullptr;
 namespace Core {
 
 Emulator::Emulator() {
-    // Initialize NT API functions and set high priority
+    // Initialize NT API functions, set high priority and disable WER
 #ifdef _WIN32
     Common::NtApi::Initialize();
     SetPriorityClass(GetCurrentProcess(), ABOVE_NORMAL_PRIORITY_CLASS);
+    SetErrorMode(SetErrorMode(0) | SEM_NOGPFAULTERRORBOX);
```

Comment:
SetErrorMode(GetErrorMode() | SEM_NOGPFAULTERRORBOX);

I dont see a reason for calling SetErrorMode(0)